src/cassandra/result.rs

```diff
@@ -193,6 +202,10 @@ impl<'a> Iterator for ResultIterator<'a> {
             }
         }
     }
+
+    fn size_hint(&self) -> (usize, Option<usize>) {
+        (self.1, Some(self.1))
```

Comment:
The lower bound isn't correct - after we've called `next()` the lower bound will be smaller. We should either update `self.1` in `next`, or put `0` here.

Reply:
good catch. i'll set to 0

Reply:
sounds good.